src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlClientMetaDataCollectionNames.cs

```diff
@@ -22,8 +22,8 @@ public static class SqlClientMetaDataCollectionNames
         /// <include file='..\..\..\..\..\..\..\doc\snippets\Microsoft.Data.SqlClient\SqlClientMetaDataCollectionNames.xml' path='docs/members[@name="SqlClientMetaDataCollectionNames"]/Indexes/*' />
         public static readonly string Indexes = "Indexes";
 
-        /// <include file='..\..\..\..\..\..\..\doc\snippets\Microsoft.Data.SqlClient\SqlClientMetaDataCollectionNames.xml' path='docs/members[@name="SqlClientMetaDataCollectionNames"]/Parameters/*' />
-        public static readonly string Parameters = "Parameters";
+        /// <include file='..\..\..\..\..\..\..\doc\snippets\Microsoft.Data.SqlClient\SqlClientMetaDataCollectionNames.xml' path='docs/members[@name="SqlClientMetaDataCollectionNames"]/ProcedureParameters/*' />
+        public static readonly string ProcedureParameters = "ProcedureParameters";
 
         /// <include file='..\..\..\..\..\..\..\doc\snippets\Microsoft.Data.SqlClient\SqlClientMetaDataCollectionNames.xml' path='docs/members[@name="SqlClientMetaDataCollectionNames"]/ProcedureColumns/*' />
         public static readonly string ProcedureColumns = "ProcedureColumns";
```

Comment:
"ProcedureColumns" is also invalid. Could you verify as well?
I got the list from here: https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/sql-server-schema-collections

Reply:
Yes. It is not valid anymore. I will change it accordingly.